recipe/envs.go

```diff
@@ -14,8 +14,9 @@ var (
 
 func populateData() map[string]string {
 	dataUpperCase, err := godotenv.Read()
+
 	// warns user about missing .env file
-	if err.Error() == "open .env: no such file or directory" {
+	if err != nil && err.Error() == "open .env: no such file or directory" {
```

Comment:
@GrayFlash we should not rely on string matching and check the type of error.

Reply:
Yes agree, will update this one. 

Actually based on a previous comment from @StewartJingga on passing the path to `secrets.yaml`, I have been thinking of adding a flag named `--config` that will allow it.

For e.g.:
`$meteor run _recipes --config secrets.yaml`
where the `--config` flag and path is optional.

And the secrets.yaml if looks like this:

```yaml
MONGO:
  USER_ID: test
  PASS: abcd1234

POSTGRES:
  USER_ID: admin
  PASS: abcd1234
```

And we can read this using the Viper package. wdyt @ravisuhag @StewartJingga ??

Reply:
@GrayFlash I am not sure if config is the right flag name for this. Maybe `--var` or `--secrets`. Or any other suggestion?

Reply:
yup config does not seem right, we could go `--var` or `-value` to be more explicit.

Reply:
+1 on `--var` cc @GrayFlash

Reply:
@StewartJingga @ravisuhag have updated the flag name to `--var`